client/src/components/Address/NewAddress.js

```diff
@@ -1,32 +1,38 @@
 import React from "react";
```

Comment:
NewAddress and EditAddress component seems to be doing the exact same, the codes are duplicated. Can we have one shared component for both Add/Edit and remove all the duplicates?

Reply:
fixed